app/services/foreman/renderer/scope/macros/kickstart.rb

```diff
@@ -0,0 +1,115 @@
+module Foreman
+  module Renderer
+    module Scope
+      module Macros
+        module Kickstart
+          include Foreman::Renderer::Errors
+          extend ApipieDSL::Module
+
+          apipie :class, desc: 'Macros to use within a kickstart template' do
+            name 'Kickstart'
+            sections only: %w[all]
+          end
+
+          # For more information about KS network directive: https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#network
+          apipie :method, "Returns a kickstart 'network' directive for a specific interface" do
```

Comment:
After a discussion at Cfgmgmt camp, let's do this as a snippet, that could be easily edited by the users.